### ISSUE TITLE
Improve color contrast

### ISSUE THREAD

**Issue by ao5357** [commercialprogression/jekyll-brochure#1]
In order for the template to have good a11y compliance, address places in the (s)CSS where the text/foreground color does not have sufficient contrast with the background.
